opentasks/src/main/java/org/dmfs/tasks/model/adapters/ColorFieldAdapter.java

```diff
@@ -90,31 +90,21 @@ public ColorFieldAdapter(String fieldName, Integer defaultValue, float darkenThr
     @Override
     public Integer get(ContentSet values)
     {
-        return darkenColor(super.get(values), mDarkenThreshold);
+        return new SmartDarkening(mDarkenThreshold, super.get(values)).argb();
     }
 
 
     @Override
     public Integer get(Cursor cursor)
     {
-        return darkenColor(super.get(cursor), mDarkenThreshold);
+        return new SmartDarkening(mDarkenThreshold, super.get(cursor)).argb();
```

Comment:
Did you consider to make this return `Color` instead of `Integer`?

Reply:
No, I didn't, since there is plan to remove `ContentSet` and current loading mechanisms, I thought it's probably not worth spending time with those structures.
Let me know if you think it's worth it. I would like to do that in a separate ticket though.

Reply:
Ok, let's do that in a separate ticket.

Reply:
Ticket added: https://github.com/dmfs/opentasks/issues/561